### DISCUSSION TITLE
Check for vulnerabilites in CPAN modules

### DISCUSSION THREAD

**Discussion by bschmalhofer** [RotherOSS/otobo#1074] [Q&A]
The console command Dev::Code::CPANAudit check for vulnerabilities using the CPAN module CPAN::Audit. However it looks like newer violations are no longer tracked in CPAN::Audit. See https://github.com/vti/cpan-audit/issues/14 . Does anybody have an idea what to do about this?

**Comment by daxim** [comment]
https://old.reddit.com/r/perl/comments/v92kct/revitalizing_cpanaudit/

**Comment by reneeb** [comment]
I'm going to release a new version of CPAN::Audit next week...

**Comment by stefanhaerter** [comment]
Nothing new here for a long time and @bschmalhofer did quite some work with CPAN::Audit recently, so I guess this discussion is outdated.
